src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+you can also set the following attributes on the viewport:
+
+* `minimum-scale`
+* `maximum-scale`
+* `user-scalable`
+
+When set, these can disable the user's ability to zoom the viewport,
+potentially causing accessibility issues.
+Therefore we would not recommend using these attributes.
+
+## Size content to the viewport {: #size-content }
+
+On both desktop and mobile devices,
+users are used to scrolling websites vertically but not horizontally;
+forcing the user to scroll horizontally or to zoom out
+in order to see the whole page results in a poor user experience.
+
+When developing a mobile site with a meta viewport tag,
+it's easy to accidentally create page content that doesn't quite fit within the specified viewport.
+For example, an image that is displayed at a width wider than the viewport
+can cause the viewport to scroll horizontally.
+You should adjust this content to fit within the width of the viewport,
+so that the user does not need to scroll horizontally.
+
+The [Content is not sized correctly for the viewport](/content-width/)
+Lighthouse audit can help you automate the process of detecting overflowing content.
+
+### Images {: #images }
+
+An image has fixed dimensions and if it is larger than the viewport will cause a scrollbar.
+A common way to deal with this problem is to give all images a `max-width` of `100%`.
+This will cause the image to shrink to fit the space it has,
+should the viewport size be smaller than the image.
+However because the `max-width`, rather than the `width` is `100%`,
+the image will not stretch larger than its natural size.
+It is generally safe to add the following to your stylesheet
+so that you will never have a problem with images causing a scrollbar.
+
+```css
+img {
+  max-width: 100%:
+  display: block;
+}
+```
+
+#### Add the dimensions of the image to the img element {: #image-dimensions }
+
+When using `max-width: 100%` you are overriding the natural dimensions of the image,
+however you should still use the `width` and `height` attributes on your `<img>` tag.
+This is because modern browsers will use this information to reserve space for the image
+before it loads in,
+this will help to avoid [layout shifts](/optimize-cls/) as content loads.
+
+### Layout {: #layout }
+
+Since screen dimensions and width in CSS pixels vary widely between devices
+(for example, between phones and tablets, and even between different phones),
+content should not rely on a particular viewport width to render well.
+
+In the past, this required setting elements used to create layout in percentages.
+In the example below, you can see a two-column layout with floated elements, sized using pixels.
+Once the viewport becomes smaller than the total width of the columns, we have to scroll horizontally
+to see the content.
+
+<figure class="w-figure">
+  <img src="./fixed-width-layout.jpg" alt="Screenshot of a two-column layout with most of the second column outside the viewport">
```

Comment:
```suggestion
  <img class="w-screenshot" src="./fixed-width-layout.jpg" alt="Screenshot of a two-column layout with most of the second column outside the viewport">
```
* This will put a little border around the image and make it look better
* https://web.dev/handbook/markup-media/#partial-screenshots